quinn/src/send_stream.rs

```diff
@@ -1,13 +1,13 @@
+use bytes::Bytes;
+use futures_channel::oneshot;
+use futures_util::{io::AsyncWrite, ready, FutureExt};
+use proto::{ConnectionError, FinishError, StreamId, Written};
 use std::{
     future::Future,
     io,
     pin::Pin,
     task::{Context, Poll},
 };
-
```

Comment:
This empty line shouldn't have been removed.

Reply:
what use blocks should remain seperate ?
can you modify it exactly how you want it ? 
```
use bytes
use futures
use proto
use std {
 ..
 ..
}
< One line space
use thiserror
< one line space
use crate::...
< one line space
```
what is the reason for the space before thiserror?


```

Reply:
Imports from `std` should be first, followed by an empty line, followed by external crate imports (if any). You've moved a bunch of non-std imports into the std block.

Reply:
@Ralith  whoops sorry, i prob deleted a newline and then run cargo fmt.
Fixed it :)